src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -171,7 +171,7 @@ internal static class LogMessages
         // public const string IDX10656 = "IDX10656:";
         public const string IDX10657 = "IDX10657: The SecurityKey provided for the symmetric key wrap algorithm cannot be converted to byte array. Type is: '{0}'.";
         public const string IDX10658 = "IDX10658: WrapKey failed, exception from cryptographic operation: '{0}'";
-        public const string IDX10659 = "IDX10659: UnwrapKey failed, exception from cryptographic operation: '{0}'";
+        public const string IDX10659 = "IDX10659: UnwrapKey failed, exception from cryptographic operation: '{0}'. If token encryption is being used, ensure the certificates are the same. See https://aka.ms/token-encryption/tutorial";
```

Comment:
we can't link to an internal wiki in a 3P library. external customers won't see the page.